pkg/platform/common/infracommon/dns.go

```diff
@@ -177,6 +195,16 @@ func (c *CommonPlatform) DeleteAppDNS(ctx context.Context, client ssh.Client, ku
 		if err != nil {
 			return err
```

Comment:
If we return here without trying to free lb IP, we might end up in a situation where this IP will just get orphaned and will never be returned back to the pool. There is an interesting dependency between deletion of load balancer IPs and DNS entries - not sure what's the simplest way to solve this.